src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.AddSub.cs

```diff
@@ -58,9 +58,7 @@ public unsafe static uint[] Add(uint[] left, uint[] right)
         }
 
         [SecuritySafeCritical]
-        private unsafe static void Add(uint* left, int leftLength,
-                                       uint* right, int rightLength,
-                                       uint* bits, int bitsLength)
+        private unsafe static void Add(uint* left, int leftLength, uint* right, int rightLength, uint* bits, int bitsLength)
```

Comment:
Please revert these changes.  The previous formatting helped to make the meaning of the parameter more clear, with each line containing the pointer and the associated length.  Please revert all similar changes that were made.